Assets/MixedRealityToolkit.Services/InputSystem/MixedRealityInputSystem.cs

```diff
@@ -868,14 +868,11 @@ public void RaiseFocusExit(IMixedRealityPointer pointer, GameObject unfocusedObj
         /// <inheritdoc />
         public void RaisePointerDown(IMixedRealityPointer pointer, MixedRealityInputAction inputAction, Handedness handedness = Handedness.None, IMixedRealityInputSource inputSource = null)
         {
+            pointer.IsFocusLocked = (pointer.Result?.Details.Object != null);
```

Comment:
>pointer.IsFocusLocked = (pointer.Result?.Details.Object != null); [](start = 12, length = 65)

This seems like a breaking change. I'm fine with it if it doesn't break our current UI but why is it necessary?

Reply:
Should not break anything, as previously we did set back the focus lock on pointer up, So should always be false at this point.